x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/es_geo_grid_source.tsx

```diff
@@ -183,6 +188,10 @@ export class ESGeoGridSource extends AbstractESAggSource implements ITiledSingle
       return 4;
     }
 
+    if (this._descriptor.resolution === GRID_RESOLUTION.SUPER_FINE) {
+      return 8;
```

Comment:
Thx for this suggestion

this makes the heatmaps more weighted to the more intense side of the color-ramp. as compared to fine/finest/coarse.

e.g. finest 
![image](https://user-images.githubusercontent.com/1833023/142693866-342e0313-dbc8-462d-a5f9-6717783824d6.png)


e.g. super fine (which has noticeable more "red" hotspots)
![image](https://user-images.githubusercontent.com/1833023/142693888-40e26d87-1b32-4034-9aae-7f6b302e29cf.png)

I think it's a fine trade-off, especially with sparser data. Super-fine will create more "no-data"-cells. I've tried it with a few datasets (see other comment), and making this smaller would make a lot of the heatmaps feel "spotty".

👍